### ISSUE TITLE
Player name truncation

### ISSUE THREAD

**Issue by jsnkuhn** [gameboy9/randoTracker#47]
Having some trouble on the zelda 2 layout with the "..." truncation. When the text is large enough to be read well we get "..."

would it be possible to either automatically reduce the font size upon overflow or allow font size control of individual player names?

**Comment by slashinfty** [comment]
In addition to players, also commentators.

**Comment by jsnkuhn** [comment]
yeah might as well cover everything. So, freeText as well.

**Comment by Datyedyeguy** [comment]
Yeah, I could see trying to scale it down so it fits.  Looked at it initially and it appears to be a bit trickier than I thought, but I can see nailing this down soon.
